pkg/operator/ceph/webhook.go

```diff
@@ -216,3 +222,47 @@ func getSecretVolume() corev1.Volume {
 	}
 	return secretVolume
 }
+
+func getTolerations(clientset kubernetes.Interface) []corev1.Toleration {
+	// Add toleration if any
+	tolerations := []corev1.Toleration{}
+	tolerationsRaw, err := k8sutil.GetOperatorSetting(clientset, controller.OperatorSettingConfigMapName, admissionControllerTolerationsEnv, "")
+	if err != nil {
+		// tolerationRaw is empty
+		logger.Warningf("toleration will not be applied. %v", err)
+		return tolerations
+	}
+	tolerations, err = k8sutil.YamlToTolerations(tolerationsRaw)
+	if err != nil {
+		logger.Warningf("failed to parse %q. %v", tolerationsRaw, err)
+		return tolerations
+	}
+	for i := range tolerations {
+		if tolerations[i].Key == "" {
```

Comment:
Why do we need to override the key and value? Seems like we should go with exactly the toleration specified.